### DISCUSSION TITLE
GitHub Sponsors — Iceland

### DISCUSSION THREAD

**Discussion by jeremybarbet** [community/community#19388] [Other Feature Feedback, Questions, & Ideas]
Hey, everything is in the title. I hope Iceland will get the GitHub Sponsors feature soon.

We are surely not as big as other countries, especially like India, but per capita, we [are](https://www.benfrederickson.com/github-developer-locations/) [rocking](https://www.researchgate.net/figure/Countries-ranked-by-number-of-OSS-developers-per-capita-top-50_tbl1_357865046), you can't ignore that! I believe we deserve the good stuff as well!

Cheers,

**Comment by melvincwng** [comment]
Hi @jeremybarbet, agree that Iceland should get Github sponsors feature soon.

Currently, Github Sponsors is only available in 38 regions.

For the full list, you can refer to it over [here](https://github.com/sponsors).

Let's hope GitHub Sponsors continue to grow and expand its availability in many other countries as well.

Cheers 🎆
